src/AppBundle/Service/EventProcessor.php

```diff
@@ -399,6 +420,26 @@ private function setItemsCreatedOrImprovedOnWikidata(EventWiki $wiki, EventWikiR
         $this->createOrUpdateEventStat('items-improved', $ret['edited']);
     }
 
+    /**
+     * If current event is category based, find out participants and add them to
+     * $this->implicitEditors.
+     * @param EventWiki $wiki
+     * @param EventWikiRepository $ewRepo
+     */
+    private function setUserCounts(EventWiki $wiki, EventWikiRepository $ewRepo): void
+    {
+        $dbName = $ewRepo->getDbNameFromDomain($wiki->getDomain());
+
+        $pageIds = $wiki->getPages();
+        $usernames = $this->getParticipantNames();
+        if ($pageIds) {
+            if (!$usernames && $this->event->getNumCategories() > 0) {
```

Comment:
lonely if? You could just use `if ($pageIds && !$usernames && $this->event->getNumCategories() > 0)`, no?

Reply:
Done.